modules/bidglassBidAdapter.js

```diff
@@ -0,0 +1,187 @@
+import * as utils from 'src/utils';
+// import {config} from 'src/config';
+import {registerBidder} from 'src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'bidglass';
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['bg'], // short code
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {BidRequest} bid The bid params to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function(bid) {
+    return !!(bid.params.adUnitId && !isNaN(parseFloat(bid.params.adUnitId)) && isFinite(bid.params.adUnitId));
+  },
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @param {validBidRequests[]} - an array of bids
+   * @return ServerRequest Info describing the request to the server.
+   */
+  buildRequests: function(validBidRequests, bidderRequest) {
+    /*
+    Use `bidderRequest.bids[]` to get bidder-dependent
+    request info.
+
+    If your bidder supports multiple currencies, use
+    `config.getConfig(currency)` to find which one the ad
+    server needs.
+    */
+
+    /*
+    Sample array entry for validBidRequests[]:
+    [{
+      "bidder": "bidglass",
+      "bidId": "51ef8751f9aead",
+      "params": {
+      "adUnitId": 11,
+      ...
+      },
+      "adUnitCode": "div-gpt-ad-1460505748561-0",
+      "transactionId": "d7b773de-ceaa-484d-89ca-d9f51b8d61ec",
+      "sizes": [[320,50],[300,250],[300,600]],
+      "bidderRequestId": "418b37f85e772c",
+      "auctionId": "18fd8b8b0bd757",
+      "bidRequestsCount": 1
+    }]
+    */
+
+    let imps = [];
+    let getReferer = function() {
```

Comment:
We already pass this information in bidderRequest. Check http://prebid.org/dev-docs/bidder-adaptor.html#referrers

Reply:
Our rules for determining and reporting referer and origins vary slightly from prebid's built-in methods. We'd prefer to use our own methods for consistency's sake across our platform.